cli/slsa-verifier/main_regression_test.go

```diff
@@ -1674,15 +1674,15 @@ func Test_runVerifyNpmPackage(t *testing.T) {
 			name:      "invalid signature provenance npm CLI",
 			artifact:  "provenance-npm-test-cli-v02-prega-invalidsigprov.tgz",
 			source:    "github.com/laurentsimon/provenance-npm-test",
-			pkgName:   PointerTo("@laurentsimon/provenance-npm-test2"),
+			pkgName:   PointerTo("@laurentsimon/provenance-npm-test"),
```

Comment:
Thanks for the new commits: it's better, but why change the pkgName here? Was it a bug?

Reply:
This unit test is supposed to check signature verification failure, not package name. So it should fail with the right package name given as input (the package name need not be different from what's in the attestation). I just realized it should be improved, it's _not_ a bug.